tests/CMakeLists.txt

```diff
@@ -1,16 +1,12 @@
 add_executable(la_test "")
 
-FetchContent_Declare(gtest GIT_REPOSITORY https://github.com/google/googletest.git)
-FetchContent_GetProperties(gtest)
 option(BUILD_GMOCK "" OFF)
 option(INSTALL_GTEST "" OFF)
 option(gtest_force_shared_crt "" ON)
 add_compile_options(-U__cpp_char8_t)
 
-if(NOT gtest_POPULATED)
-    FetchContent_Populate(gtest)
-    add_subdirectory(${gtest_SOURCE_DIR} ${gtest_BINARY_DIR})
-endif()
+FetchContent_Declare(gtest GIT_REPOSITORY https://github.com/google/googletest.git)
+FetchContent_MakeAvailable(gtest)
```

Comment:
`FetchContent_MakeAvailable` does the same as checking the populated variable, populating and then adding sub directory.